README.md

```diff
@@ -38,15 +38,8 @@ A docker image is available to run following script
 
 [![logo](https://img.shields.io/docker/pulls/jenkinsciinfra/packaging?label=jenkinsciinfra%2Fpackaging&logo=docker&logoColor=white)](https://hub.docker.com/r/jenkinsciinfra/packaging)
 
-```shell
-docker run -i -t \
-  -v $PWD:/packaging \
-  -v /gpg_password:/secrets/password \
-  -eWAR="/packaging/jenkins.war" \
-  -eGPG_KEYNAME="sandbox" \
-  -eGPG_PASSPHRASE_FILE="/secrets/password" \
-  -w /packaging jenkinsciinfra/packaging:latest make deb`
-```
+Run `./bldenv.sh` to get a shell in the official Docker image for this repository.
```

Comment:
what does bld stand for? if it's build could we rename to `build-env` ?

Reply:
Feedback about variable naming is consistently the lowest-quality code review feedback that I receive. This particular script is named after the `bldenv` script in [illumos](https://github.com/illumos/illumos-gate/blob/master/usr/src/tools/scripts/bldenv), which has been performing a similar function for roughly three decades.

Reply:
What do you think about adding a comment that would help the reader to know that? I have no idea what bldenv and what illumos is, and it would help to review the PR AND self document the intent here.

Reply:
Question: is there a reason to use a shell script against a docker-compose item (which is shipped by default on docker now)?

Reply:
Deleted `bldenv.sh`.